x-pack/plugins/security_solution/common/detection_engine/get_query_filter.ts

```diff
@@ -110,27 +110,35 @@ export const buildEqlSearchRequest = (
     exceptionFilter = buildExceptionFilter(exceptionQueries, indexPattern, config, true, 1024);
   }
   const indexString = index.join();
+  const requestFilter: unknown[] = [
```

Comment:
I believe you could use `EsRangeFilter` here in place of the `unknown`:

```suggestion
  const requestFilter: EsRangeFilter[] = [
```

Reply:
Actually, since you're pushing the `bool` object below as well, not sure what the recommended generic Filter type that should be used here...